docs/references/http-websocket-apis/public-api-methods/ledger-methods/ledger.md

```diff
@@ -105,7 +105,7 @@ The response follows the [standard format][], with a successful result containin
 | `ledger.close_time_resolution` | Number  | Ledger close times are rounded to within this many seconds. |
 | `ledger.closed`                | Boolean | Whether or not this ledger has been closed. |
 | `ledger.ledger_hash`           | String  | Unique identifying hash of the entire ledger. |
-| `ledger.ledger_index`          | String  | The [Ledger Index][] of this ledger, as a quoted integer. |
+| `ledger.ledger_index`          | [API v1][]: String <br> [API v2][]: Number  | The [Ledger Index][] of this ledger, as a quoted integer. |
```

Comment:
If it's a Number, then I don't think it would be a quoted integer.

Reply:
Fixed this error in #2984